static/src/stylesheets/module/_badging.scss

```diff
@@ -18,25 +16,50 @@
     width: 100%;
 }
 
-.content__labels--us-election {
-    .content__section-label,
-    .content__series-label {
+/* Container Rules
+   ========================================================================== */
+.fc-container__header--voices-of-america {
+    .badge-slot {
+        width: gs-span(1.5);
+        float: left;
+
+        @include mq(leftCol) {
+            width: gs-span(2);
+        }
+
+        @include mq(wide) {
+            width: gs-span(2.5);
+        }
+    }
+
+    ~ .fc-container__header__description {
         float: none;
     }
+
+    .fc-container__title__text {
+        display: none;
+    }
 }
 
-.content__labels--aus-election {
+/* Content Rules
+   ========================================================================== */
+.content__head--is-badged {
     .content__section-label,
     .content__series-label {
         float: none;
     }
 }
 
-/* AUS Election Liveblog Image
-   ========================================================================== */
-.content__head--election {
+.content__head--voices-of-america {
+    .badge-slot {
+        display: none;
+    }
+}
 
-    &.aus-election {
+/* Liveblog Rules
+   ========================================================================== */
+.content--liveblog {
```

Comment:
Hey Sam, adding this `.content--liveblog` wrapper to around the styles below applies padding changes in `.content__header .gs-container` to all liveblog headers, not just the election ones.